Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "elastic-elgamal"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
I prefer to bump the crate version right before publishing; makes publishing less confusing, IMO.

Reply:
I've changed the version back to `0.2.1` since it seemed to be causing one of the CI failures.